### PR TITLE
4632 show delivery quantity estimation

### PR DESCRIPTION
**Pull request by samMcCarthyBJSS**
Story [AB#4632](https://buyingcatalog.visualstudio.com/c5f97979-5b03-4d10-ba8d-871d0526b408/_workitems/edit/4632)
Task [AB#7717](https://buyingcatalog.visualstudio.com/c5f97979-5b03-4d10-ba8d-871d0526b408/_workitems/edit/7717)
image

### CODE REVIEW

File: app/pages/sections/catalogue-solutions/order-item/template.test.js [hunk 1]
```diff
@@ -0,0 +1,235 @@
...
+        expect(labels[0].attribs.for).toEqual(`${manifest.questions.plannedDate.id}-day`);
+        expect(labels[0].children[0].data.trim()).toEqual('Day');
+        expect(labels[1].attribs.for).toEqual(`${manifest.questions.plannedDate.id}-month`);
+        expect(labels[1].children[0].data.trim()).toEqual('Month');
+        expect(labels[2].attribs.for).toEqual(`${manifest.questions.plannedDate.id}-year`);
+        expect(labels[2].children[0].data.trim()).toEqual('Year');
+      });
+    }));
+
+
```
Comment (alibombaye) [inline app/pages/sections/catalogue-solutions/order-item/template.test.js:103]: extra blank line

File: app/pages/sections/catalogue-solutions/order-item/template.test.js [hunk 2]
```diff
@@ -0,0 +1,235 @@
...
+          ],
+          expandableSection: {
+            dataTestId: 'view-section-estimation-period-id',
+            title: 'What period should I enter?',
+            innerComponent: 'This should be based on how you estimated the quantity you want to order.',
+          },
+        },
+      },
+    };
+    it('should render the "Estimation period" radio button options component', componentTester(setup, (harness) => {
```
Comment (alibombaye) [inline app/pages/sections/catalogue-solutions/order-item/template.test.js:190]: add a blank like after 189 please

File: app/pages/sections/catalogue-solutions/order-item/ui-tests/general.ui.test.js [hunk 3]
```diff
@@ -0,0 +1,276 @@
...
+
+//   const selectRecipientRadioOptions = Selector('[data-test-id="question-selectRecipient"]');
+//   const firstRecipient = selectRecipientRadioOptions.find('input').nth(0);
+//   const button = Selector('[data-test-id="continue-button"] button');
+
+//   await t
+//     .click(firstRecipient)
+//     .click(button)
+//     .expect(getLocation()).eql('http://localhost:1234/order/organisation/order-id/catalogue-solutions/newsolution');
+// });
```
Comment (alibombaye) [inline app/pages/sections/catalogue-solutions/order-item/ui-tests/general.ui.test.js:276]: remove comments

File: app/pages/sections/catalogue-solutions/order-item/ui-tests/general.ui.test.js [hunk 4]
```diff
@@ -0,0 +1,276 @@
...
+
+test('should render additionalAdvice', async (t) => {
+  await pageSetup();
+  await t.navigateTo(pageUrl);
+
+  const additionalAdvice = Selector('[data-test-id="date-field-input"] span.nhsuk-hint');
+
+  await t
+    .expect(additionalAdvice.exists).ok()
+    .expect(await extractInnerText(additionalAdvice)).eql(content.plannedDateQuestions[0].additionalAdvice);
```
Comment (alibombaye) [inline app/pages/sections/catalogue-solutions/order-item/ui-tests/general.ui.test.js:134]: bit confused as to how these are passing. We have changed the manifest so this should be 
`content.questions.plannedDate.additionalAdvice` ??

Comment (samMcCarthyBJSS) [reply app/pages/sections/catalogue-solutions/order-item/ui-tests/general.ui.test.js:134]: ah they hadn't passed it was only the unit tests I ran will fix them up

**Pull request merged**